server/src/main/scala/drt/server/feeds/bhx/BHXFeed.scala

```diff
@@ -26,59 +26,30 @@ import scala.xml.{Node, NodeSeq}
 object BHXFeed {
   val log: Logger = LoggerFactory.getLogger(getClass)
 
-  def apply(bhxLiveFeedUser: String, bhxSoapEndpoint: String)(implicit actorSystem: ActorSystem): Source[ArrivalsFeedResponse, Cancellable] = {
+  var initialRequest = true
 
-    val client = BHXClient(bhxLiveFeedUser, bhxSoapEndpoint)
-    val pollFrequency = 5 minutes
-    val initialDelayImmediately: FiniteDuration = 1 milliseconds
-    val tickingSource: Source[ArrivalsFeedResponse, Cancellable] = Source.tick(initialDelayImmediately, pollFrequency, NotUsed)
+  def apply(client: BHXClientLike, pollFrequency: FiniteDuration, initialDelay: FiniteDuration)(implicit actorSystem: ActorSystem): Source[ArrivalsFeedResponse, Cancellable] = {
+
+    var initialRequest = true
```

Comment:
This is declared twice. Probably only want one of them

Reply:
👍